trade_remedies_public/templates/v2/registration/how_to_get_account.html

```diff
@@ -0,0 +1,19 @@
+{% extends "v2/govuk/base_with_form.html" %}
```

Comment:
no need to extend from base_with_form, can just use base.html instead
```suggestion
{% extends "v2/govuk/base.html" %}
```